src/sentry/snuba/metrics/extraction.py

```diff
@@ -549,7 +551,8 @@ def _deep_sorted(value: Union[Any, Dict[Any, Any]]) -> Union[Any, Dict[Any, Any]
 TagsSpecsGenerator = Callable[[Project, Optional[str]], List[TagSpec]]
 
 
-def failure_rate_tag_spec(_1: Project, _2: Optional[str]) -> List[TagSpec]:
+def failure_tag_spec(_1: Project, _2: Optional[str]) -> List[TagSpec]:
```

Comment:
It's a shared tag spec between `failure_count` and `failure_rate`, thus, renaming it to avoid using `count` or `rate` in the name.